include/irr/asset/IAsset.h

```diff
@@ -58,11 +58,13 @@ class IAsset : virtual public core::IReferenceCounted
         ET_PIPELINE_LAYOUT = 1u<<15,
         //! asset::ICPUBufferView
         ET_BUFFER_VIEW = 1u<<16,
+        //! asset::ICPUSampler
+        ET_SAMPLER = 1u<<17,
```

Comment:
can you reorder those bitfields so they make sense?

Reply:
@Crisspl TODO.

Reply:
resolved